src/user.ts

```diff
@@ -90,6 +90,16 @@ export class UserModule {
         }, (err: ApiResponseError) => Promise.reject(err));
     }
 
+    /**
+     * Gets current user's profile
+     * @method getCurrentProfile
+     * @return {Promise} resolves with account setting object if user logged in, false otherwise
+     */
+    public getCurrentProfile() {
+        return this.xhr.get('/gdc/account/profile/current')
+            .then(r => r.getData().accountSetting);
```

Comment:
This should be more similar to this approach 

https://github.com/gooddata/gooddata-js/blob/3e86de798f6c2541bf7200adcad9161dc528edc6/src/execution/execute-afm.ts#L51

Reply:
https://jira.intgdc.com/browse/BB-1196